packages/drivers/odsp-driver/src/odspSnapshotParser.ts

```diff
@@ -64,6 +64,9 @@ export function convertOdspSnapshotToSnapsohtTreeAndBlobs(
     const val: ISnapshotContents = {
         blobs: blobsWithBufferContent,
         ops: odspSnapshot.ops?.map((op) => op.op) ?? [],
+        // This statement cannot use optional chaining safely because if it shortcirrcuits an undefined error will be
+        // thrown
+        // eslint-disable-next-line @typescript-eslint/prefer-optional-chain
         sequenceNumber: odspSnapshot.trees && (odspSnapshot.trees[0]).sequenceNumber,
```

Comment:
I don't understand what this comment means, would this not be equivalent?

```suggestion
        sequenceNumber: odspSnapshot.trees?.[0].sequenceNumber,
```

Reply:
I can't reproduce the failure I was getting, so <shrug>. I'll update.